tests/utils.go

```diff
@@ -518,8 +520,8 @@ func WaitVirtualMachineDeleted(client kubecli.KubevirtClient, namespace, name st
 }
 
 func NewDataVolumeForFedoraWithGuestAgentImage(dataVolumeName string, storageClass string) *cdiv1.DataVolume {
```

Comment:
you should probably change the name of the function :)

Reply:
;) thanks, did not notice that

Reply:
done